### PR TITLE
Support Rails5

### PR DESCRIPTION
**Pull request by takkanm**
include #115 commit. thanks @rdlu.

### CODE REVIEW

File: spec/dummy/config/environments/test.rb [hunk 1]
```diff
@@ -40,4 +47,9 @@
 
   # Raises error for missing translations
   # config.action_view.raise_on_missing_translations = true
+
+  # Time columns will become time zone aware in Rails 5.1.
+  if Rails.version.split('.').first.to_i >= 5
```
Comment (ursm) [inline spec/dummy/config/environments/test.rb]: I think there is a way better than this. For example it's to use `Rails::VERSION::MAJOR`.

File: app/controllers/adhoq/executions_controller.rb [hunk 2]
```diff
@@ -44,7 +44,17 @@ def async_execution?
...
 
     def query_parameters
-      params[:parameters] || HashWithIndifferentAccess.new
+      if params[:parameters]
+        if params[:parameters].instance_of?(Hash)
+          params[:parameters]
+        else
+          # for after Rails5
+          params[:parameters].permit!
+          params[:parameters].to_h
```
Comment (ursm) [inline app/controllers/adhoq/executions_controller.rb]: Can't these two lines be written this?

``` ruby
params[:paremeters].to_unsafe_hash
```

(Only this condition has returned pure Hash, but is that OK?)

File: spec/dummy/config/environments/test.rb (hunk 1 above)
Comment (takkanm) [inline spec/dummy/config/environments/test.rb]: thanks. I forgot this API. 😨

File: app/controllers/adhoq/executions_controller.rb (hunk 2 above)
Comment (takkanm) [inline app/controllers/adhoq/executions_controller.rb]: Thanks. I didn't know this method.

**Pull request merged**